examples/ios/swift/Migration/URLExtension.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  URLExtension.swift
+//  Migration
+//
+//  Created by Dominic Frei on 28/01/2021.
+//  Copyright © 2021 Realm. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+extension URL {
```

Comment:
I would avoid extensions like this as it reduces readability, but if you can't find a better way to do this, I would move it into the AppDelegate file.

Reply:
I made another suggestion and moved it to `extension RealmVersion`.